tests/helpers/WPCOM_API_Helper/test-wpcom-api-helper.php

```diff
@@ -0,0 +1,49 @@
+<?php
+declare(strict_types=1);
+
+namespace Team51\Test;
+
+use PHPUnit\Framework\TestCase;
+
+use Team51\Helper\WPCOM_API_Helper;
+
+/**
+ * Tests for the WPCOM_API_Helper class.
+ */
+class WPCOM_API_Helper_Test extends T51TestCase {
+
+	/**
+	 * WPCOM_API_Helper instance.
+	 *
+	 * @var WPCOM_API_Helper
+	 */
+	public $wpcom_api_helper;
+
+	public function __construct() {
+		parent::__construct();
+		$this->wpcom_api_helper = new WPCOM_API_Helper();
+	}
+
+	/**
+	 * Test assembly of request URL.
+	 *
+	 * @covers WPCOM_API_Helper::get_request_url()
+	 */
+	public function test_get_request_url_adds_prefix_with_integer() {
+		$expected = 'https://public-api.wordpress.com/rest/v1.1/sites/123';
+		$actual   = $this->invoke_method( $this->wpcom_api_helper, 'get_request_url', array( 'sites/123' ) );
```

Comment:
How does this differ in practice from just 

```suggestion
		$actual   = $this->wpcom_api_helper->get_request_url( 'sites/123' );
```

Reply:
This would give you an error since that method is private. The invoke method changes that with the `$method->setAccessible( true );` in the ReflectionClass. 
Almost all of the methods in our commands, and most of the helpers, are protected or private. To make testing of these easier I decided to add the `invoke_method` as part of the extended TestCase to avoid having to re-write the method every time.

Reply:
Got it!  That makes sense.